library/src/shared/constants/constants.ts

```diff
@@ -48,4 +48,6 @@ export class Constants {
     fiat: 'USD'
   };
   static DEFAULT_COMPONENT = 'price-list';
+  static PERSONA_SCRIPT_SRC =
+    'https://cdn.withpersona.com/dist/persona-v4.6.0.js';
```

Comment:
Is this used in code anywhere?

Reply:
Now that we've merged the component it is.